core/pom.xml

```diff
@@ -12,6 +12,11 @@
   <packaging>jar</packaging>
   <version>1.0-SNAPSHOT</version>
   <dependencies>
+    <dependency>
+      <groupId>com.es.phoneshop</groupId>
+      <artifactId>phoneshop-demodata</artifactId>
```

Comment:
Does it running fine on your local? For me app is not running with that circular dependency here

Reply:
It actually didn't, but I added this dependency for tests and tests were running fine and I didn't check the overall application. I'll fix that, thanks for your remark.